src/localizer.cpp

```diff
@@ -344,12 +344,9 @@ bool Localizer::loadLanguage(const QString& requestedLocalCode) {
     localeCode = systemLanguageCode();
   }
 
-  double completeness = m_translationCompleteness.value(localeCode, 0);
-  if (completeness < 1) {
```

Comment:
Am I understanding correctly that we are removing the check for 100% language completion and always loading our fallback languages no matter what? 

Also are there any drawbacks to loading a fallback language even when a language (i.e. english.. do we even generate a fallback language for english) is truly at 100% (including new strings)?

Reply:
That is a correct understanding.

I think it would be pretty ugly to try to detect if there were new strings causing us to falsely report 100% - and this false 100% report was why this bug existed. I'm not aware of downsides of loading backup languages, but I'm also not well-versed in this. Do you (or anyone else looking at this ticket) know of any potential downsides that I could investigate?